lib/kepler_web/templates/user_session/new.html.heex

```diff
@@ -0,0 +1,27 @@
+<h1>Log in</h1>
+
+<.form let={f} for={@conn} action={Routes.user_session_path(@conn, :create)} as={:user}>
+  <%= if @error_message do %>
+    <div class="alert alert-danger">
+      <p><%= @error_message %></p>
+    </div>
+  <% end %>
+
+  <%= label f, :email %>
+  <%= email_input f, :email, required: true %>
+
+  <%= label f, :password %>
+  <%= password_input f, :password, required: true %>
+
+  <%= label f, :remember_me, "Keep me logged in for 60 days" %>
```

Comment:
While I realize it makes the semantic HTML a little weird, having the label before the checkbox this way makes the checkbox placed after the label, when it should probably be before. </nitpick>

Reply:
Yeah, I'm just going to leave that until we redo all the pages with tailwind if that's alright.

Reply:
Yeah that's fine.